### PR TITLE
{tools}[GCCcore/11.2.0] hunspell v1.7.1

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/h/hunspell/hunspell-1.7.0-GCCcore-11.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Contribution from the Crick HPC team
+# uploaded by J. Sassmannshausen
+
+easyblock = 'ConfigureMake'
+
+name = 'hunspell'
+version = '1.7.0'
```
Comment (boegel) [inline easybuild/easyconfigs/h/hunspell/hunspell-1.7.0-GCCcore-11.2.0.eb:7]: @sassy-crick v1.7.1 was released recently, let's bump to that?

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/h/hunspell/hunspell-1.7.0-GCCcore-11.2.0.eb (hunk 1 above)
Comment (sassy-crick) [reply easybuild/easyconfigs/h/hunspell/hunspell-1.7.0-GCCcore-11.2.0.eb]: Done. There was no particular reason for not using the latest version.

**Comment by zao** [comment]
Test report by @zao
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
eb-u2204 - Linux Ubuntu 22.04, x86_64, AMD Ryzen 9 3900X 12-Core Processor, Python 3.10.4
See https://gist.github.com/c002947a97529f785f089f6b1d0ec321 for a full test report.

**Comment by zao** [comment]
Build above fails on a missing `autopoint` executable, supposedly from GNU gettext (or gnu-lib if that's vendored).

**Comment by sassy-crick** [comment]
> Build above fails on a missing `autopoint` executable, supposedly from GNU gettext (or gnu-lib if that's vendored).

Done. That has slipped through.
Question: how are we treating `gawk` ? If I want to add it, I need to change the toolchaine from `GCCcore` to `GCC`. Would that break things later on?

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3144.skitty.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/f17a83b8e33153c082a051bd7e868c97 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @sassy-crick!

**Pull request merged**